src/gtest/test_deprecation.cpp

```diff
@@ -36,7 +36,7 @@ class DeprecationTest : public ::testing::Test {
 
     virtual void TearDown() {
         fRequestShutdown = false;
-        mapArgs["-disabledeprecation"] = "";
+        mapArgs.clear();
```

Comment:
I'm trying to imagine how this might interfere with other tests. But we don't set any arguments (that would end up in here) when starting up the test binaries, so I guess it's fine?

Reply:
I was running under the assumption that these tests were self contained, if that is not true I'll clear each flag manually.

Reply:
They *should* be. So actually, it's a good idea to do this, and we should fix any broken tests it creates.